src/official-account/simple-unirest.ts

```diff
@@ -9,7 +11,7 @@ export interface FileInfo {
 type RequestType = 'json' | 'html'
 
 interface UnirestRequest<T> extends Promise<{ body: T }> {
-  attach : (formName: string, buf: Buffer, info?: FileInfo) => UnirestRequest<T>
+  attach : (formName: string, buf: Buffer | ReadStream, info?: FileInfo) => UnirestRequest<T>
```

Comment:
Add node ReadStream type to attach interface function.

Reply:
LGTM